airbyte-webapp/src/views/Connector/ServiceForm/useAuthentication.tsx

```diff
@@ -202,9 +208,22 @@ export const useAuthentication = (): AuthenticationHook => {
     [advancedAuth, isAuthButtonVisible, legacyOauthSpec]
   );
 
+  const hasAuthFieldValues: boolean = useMemo(() => {
+    if (
+      implicitAuthFieldPaths.some((path) => {
+        return get(values, path) !== undefined;
```

Comment:
generally not a fan of using lodash, but this seemed a lot cleaner than parsing the string and traversing the object more manually.  happy to go that direction if it seems better to whomever reviews, though.

Reply:
We can actually use getIn here which is from formik and already used across this file, which does the same. So we wouldn't need to pull in another library in this place.

Reply:
Ah thank you!